prinz/src/main/scala/pl/touk/nussknacker/prinz/model/proxy/ProxiedHttpInputModelBuilder.scala

```diff
@@ -0,0 +1,34 @@
+package pl.touk.nussknacker.prinz.model.proxy
+
+import pl.touk.nussknacker.engine.util.SynchronousExecutionContext.ctx
+import pl.touk.nussknacker.engine.api.typed.typing.{Typed, TypingResult}
+import pl.touk.nussknacker.prinz.model.{Model, SignatureName, SignatureType}
+import pl.touk.nussknacker.prinz.util.http.AsyncRestJsonClient
+
+import scala.concurrent.Future
+
+class ProxiedHttpInputModelBuilder(model: Model) extends ProxiedInputModelBuilder(model) {
+
+  def proxyHttpGet(paramName: String, path: String): this.type = {
+    val httpClient = new AsyncRestJsonClient(path)
+    val signatureName = SignatureName(paramName)
+    proxyParam(paramName) { modelInputParamMetadata =>
+      val signature = modelInputParamMetadata.modelMetadata.signature
+      signature.getInputValueType(signatureName) match {
+        case Some(paramType) => httpGetAndDeserialize(paramType, httpClient)
+        case None => Future(Unit)
```

Comment:
Why do we return Future(Unit) here?

Reply:
it works here in a way that when some input param name is specified in this method as parameter and then checked in signature for existence  (with getInputValueType method), we do http request using httpGetAndDeserialize which returns the Future[AnyRef] and the value of the same type should be returned in the other case (when input param name not exists in signature) - we do return this unit which can be treated as void action with proper type as this value will never be used (because it does not exist in signature)